packages/aa/src/index.js

```diff
@@ -144,6 +152,15 @@ function walkDependencyTreeForBestLogicalPaths({
     }
   } while (currentLevelTodos.length > 0)
 
+  for (const [
+    packageDir,
+    logicalPath,
+  ] of preferredPackageLogicalPathMap.entries()) {
+    if (isSymlink(packageDir)) {
```

Comment:
is this faster than just _always_ getting the real path?

Reply:
I assumed real path is much more work, but maybe only in case the thing is a link or a link to a link...
Gonna need to measure.

Reply:
Covered with a microbenchmark. isSymlink is almost an order of magnitude faster on both symlinks and not. 
```
 results: {
          'isSymlink(notsymlink)': 16.870908999999983,
          'isSymlink(symlink)': 27.16377,
          'realpathSync(notsymlink)': 142.83877799999993,
          'realpathSync(symlink)': 237.59675600000003,
        },
```

If that's no longer the case or is not the case on one of the node versions, tests will warn us